processor/processor.go

```diff
@@ -940,6 +1236,12 @@ func (proc *HandleT) crashRecover() {
 			}
 			statusList = append(statusList, &status)
 		}
-		proc.gatewayDB.UpdateJobStatus(statusList, []string{gateway.CustomVal}, nil)
+		db.UpdateJobStatus(statusList, []string{customVal}, nil)
 	}
 }
+
+func (proc *HandleT) crashRecover() {
+	markExecJobsToFailed(proc.gatewayDB, gateway.CustomVal)
+	markExecJobsToFailed(proc.errorDB, "")
```

Comment:
We are setting customVal to destType for procErrDb. But querying with an empty string here. Will this work?

Reply:
we are storing all the errors combined to s3, not per destination. The read query from proc_err_jobs also does with empty string for customVal